app/models/player_profile.rb

```diff
@@ -4,6 +4,8 @@ class PlayerProfile < ApplicationRecord
   has_many :recruiter_profiles, through: :prospects
   has_many :player_stats
 
+  enum status: ["active", "inactive"]
```

Comment:
@katiekeel 

I like the idea of using enum status for active/inactive status. What do you think about doing a background worker at some point that would change the enum status based on date?

Reply:
@mdevoe12 

That would be a great idea. Since we're out of recruitment season right now, implementing the background worker would currently show all the recruits as inactive, which would be pretty useless for the demo of the actual recruitment contact process. It's something we can implement after Thursday though.

Reply:
Sounds good. Merging.

🌮